2018/faq-qaf.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+order: 10
+title: "2018"
+subtitle: "FAQ | QAF"
+titleDisplay: "FAQ"
+excerpt:
+venueId:  
+date:
+startTime:
+endTime:
+---
+
+We have a slowly-growing list of Frequently asked questions/Questions to ask frequently, if you don't see yours below, send us an email
+
+### Are the venues accessible?
+
+We are committed to addressing barriers to access. So far we've confirmed a Conference and Sprint space that are entered by ramp and elevator, with accessible washrooms.
+
+### What is the deal with project sprints?
+
+Project sprints are a great way to work together to learn and create Do It With Others (DIWO)-style. The [Sprints](/2018/sprints/) page has a list of projects and details for proposing your own
+
+### I am traveling from out of town, how can I stay in Toronto affordably? (a.k.a. What are Community Billets?)
+
+We do not have a discount rate with a hotel, however we have arranged for a limited number of spots for those looking for a place to stay, check out the [community billets list](#) to coordinate!
```

Comment:
Need to think through this a little more -- would an open ethercalc spreadsheet work? I think we could have contact name, description of avail, intersection and then just the email part becomes ??? (set up billets@ournetworks.ca? have every billet person on it and they can respond as they should?)